katacodacompolyversepolymorphic-linux-install-centos-container/subscribe.md

```diff
@@ -1,7 +1,6 @@
-Polymorphic Linux is distributed as drop-in replacement packages for all the existing packages on an existing Linux
-host, by pulling them from a Polyverse "mirror".
+Polymorphic Linux is distributed as drop-in replacement packages for all of the existing packages on your host. To use, simply point your package repository to the Polyverse repositories instead of a community run “mirror”. From then on, Polymorphic Linux operates identically to standard Linux.
 
-We provide a simple script to subscribe to the mirrors, found on the
-[free trial page](https://polyverse.io/polymorphic-linux-installation-guide/).
+We provide a simple script to subscribe to the mirrors, this command can found on the
```

Comment:
...this command can **be** found...

Reply:
I just tried to make the change and add "be" did it work?